src/content/developers/tutorials/erc721-vyper-annotated-code/index.md

```diff
@@ -0,0 +1,731 @@
+---
```

Comment:
Mind just changing the filename to hyphenate erc-721?
`/erc-721-vyper-annotated-code/index.md`